ui/lib/pki/addon/components/pki-generate-csr.ts

```diff
@@ -82,7 +83,7 @@ export default class PkiGenerateCsrComponent extends Component<Args> {
 
   @task
   @waitFor
-  *save(event: Event): Generator<Promise<boolean | PkiActionModel>> {
+  *save(event: Event): Generator<Promise<boolean | Model>> {
```

Comment:
With the changes [here](https://github.com/hashicorp/vault/pull/29702/files#diff-aecc354ea8358e39de06f7d145f0b7d6a909797d045b8465da58920e82a003cdR6-R9) - shouldn't we be able to keep this `PkiActionModel`?

Reply:
I thought so too but it complains about wanting `Model` if I revert it to `PkiActionModel` 🤷.